metashare/api/jobs.py

```diff
@@ -184,17 +195,88 @@ def create_branches_on_github_then_create_scratch_org(
             repo_branch=commit_ish,
             project_path=repo_root,
         )
+        get_unsaved_changes(scratch_org, should_save=False)
 
 
 create_branches_on_github_then_create_scratch_org_job = job(
     create_branches_on_github_then_create_scratch_org
 )
 
 
+def get_unsaved_changes(scratch_org, should_save=True):
```

Comment:
Now that this function is being called multiple places to update the changes (e.g. after a commit), it is triggering many unnecessary `SCRATCH_ORG_UPDATED` WS events.